GPU/Common/ShaderId.h

```diff
@@ -49,6 +49,7 @@ enum VShaderBit : uint8_t {
 	VS_BIT_LIGHT3_ENABLE = 55,
 	VS_BIT_LIGHTING_ENABLE = 56,
 	VS_BIT_WEIGHT_FMTSCALE = 57,  // only two bits
+	VS_BIT_POINT_SIZE = 58,  // 1 bit
```

Comment:
Doesn't this overlap with VS_BIT_WEIGHT_FMTSCALE?

-[Unknown]

Reply:
Yeah, oops.